components/script/dom/document.rs

```diff
@@ -3438,7 +3436,7 @@ impl DocumentMethods for Document {
 
     // https://html.spec.whatwg.org/multipage/#dom-document-location
     fn GetLocation(&self) -> Option<Root<Location>> {
-        self.browsing_context().map(|_| self.location.or_init(|| Location::new(&self.window)))
+        self.browsing_context().map(|_| self.window.Location())
```

Comment:
I believe the document needs to be checked whether it's [`fully active`](https://html.spec.whatwg.org/multipage/browsers.html#fully-active), instead of checking whether there's a browsing context.

Reply:
Fixed.